core/controllers/reader_test.py

```diff
@@ -1189,6 +1189,221 @@ def test_remove_collection_from_incomplete_list_handler(self):
                 self.user_id), [])
 
 
+class StorePlaythroughHandlerTest(test_utils.GenericTestBase):
+    """Tests for the handler that records playthroughs."""
+
+    def setUp(self):
+        super(StorePlaythroughHandlerTest, self).setUp()
+        self.exp_id = '15'
+
+        self.login(self.VIEWER_EMAIL)
+        self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
+        exp_services.delete_demo(self.exp_id)
```

Comment:
Does this need to be run for the very first test, or is it to clean-up just in case one a demo already exists? In that case, maybe it belongs in `def tearDown`?

Reply:
Actually, its not required. Ive removed it.

Reply:
OK, please commit that then!